### PR TITLE
Update measured launch status checks

### PR DESCRIPTION
**Pull request by dpsmith**
This fixes the existing pcrdiff utils to provide reports on PCR changes and adds a utility to report the recovery key scheme that is in use on the device.

OXT-1062

Signed-off-by: Daniel P. Smith <dpsmith@apertussolutions.com>

### CODE REVIEW

File: recipes-openxt/openxt/openxt-measuredlaunch_1.0.bb [hunk 1]
```diff
@@ -6,19 +6,22 @@ inherit xenclient
 
 SRC_URI = " \
     file://ml-functions \
-    file://seal-system \
+    file://recovery-method \
+
```
Comment (jean-edouard) [inline recipes-openxt/openxt/openxt-measuredlaunch_1.0.bb]: Empty line without a '\', won't parse, please test code before submitting pull requests.

Comment (dpsmith) [inline recipes-openxt/openxt/openxt-measuredlaunch_1.0.bb]: The code(script) was tested before submitting. The recipe change was minor and inspection is sufficient.

Comment (jean-edouard) [inline recipes-openxt/openxt/openxt-measuredlaunch_1.0.bb]: Obviously not.

**Comment by jean-edouard** [comment]
Built here: http://openxt-builder.ainfosec.com:8010/builders/openxt/builds/1034

**Comment by jean-edouard** [comment]
Merging soon.

**Pull request merged**